examples/md-flexible/scripts/compareOptimumCongfiguration.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+
+import sys
+import re
+from numpy import double
+
+# THIS SCRIPT NEEDS AT LEAST PYTHON 3.8.
+# However, lesser version will probably fail due to invalid syntax instead of this assertion
+
+# ---------------------------------------------- Input ----------------------------------------------
+
+# help message
+for arg in sys.argv[1:]:
+    if "--help" in arg:
+        print("Usage: ./compareOptimumConfiguration.py [path/To/mdFlex/std.out ...]")
+        print("The script need at least to input files, where the first file is the base to which the other files are"
+              " going to be compared to.")
+        print("If no or not enough input files are given the script does not work.")
+        exit(0)
+
+# take all input files as source for a plot
+if len(sys.argv) > 2:
+    datafiles = sys.argv[1:]
+else:
+    print("Error: No or wrong input given! ./compareOptimumConfiguration.py --help to see what is needed.")
+    sys.exit(-1)
+
+# ---------------------------------------------- Script ---------------------------------------------
+
+comparisonBase = []
+comparisonFiles = []
+
+# collect data from files
+for datafile in datafiles:
+
+    # parse file
+    with open(datafile, 'r') as file:
+
+        # variables for data collection
+        configurationsSelected = []
+        configurationTest = {}
+
+        regexSelectedConfiguration = '.* Selected Configuration +({.*})'
+        regexCollectedTimes = '.* Collected times for +({.*})..*\[(.*)\].*: *([0-9]+)'
```

Comment:
please include some error handling when nothing was found. This can happen for example if
a) the simulation run was too short to yield a tuning
b) wrong log level
c) totally wrong file